tests/test_time_machine.py

```diff
@@ -434,6 +434,36 @@ def test_destination_datetime_tzinfo_zoneinfo_no_orig_tz():
         with time_machine.travel(dest):
             assert time.tzname == ("EAT", "EAT")
 
+        dest2 = LIBRARY_EPOCH_DATETIME.replace(tzinfo=ZoneInfo("UTC"))
+
+        with time_machine.travel(dest2):
+            assert time.tzname == ("UTC", "UTC")
+
+        assert time.tzname == orig_tzname
+
+
+def test_destination_datetime_tzinfo_datetime_timezone_utc_no_orig_tz():
+    with change_local_timezone(None):
+        orig_tzname = time.tzname
+        dest = LIBRARY_EPOCH_DATETIME.replace(tzinfo=dt.timezone.utc)
+
+        with time_machine.travel(dest):
+            assert time.tzname == ("UTC", "UTC")
+
+        assert time.tzname == orig_tzname
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 11), reason="datetime.UTC was introduced in Python 3.11"
```

Comment:
I could skip this either if `datetime.UTC` attribute is missing or using Python version check (as shown). Let me know if you want me to change this.

Reply:
Use a python version check - then I’ll spot and remove it in the future when dropping old python version support.

Reply:
Great! That's what I thought as well and so I chose to go with Python version checking.